ui/packages/components/.storybook/preview.tsx

```diff
@@ -8,13 +9,13 @@ const preview: Preview = {
   decorators: [
     (Story) => {
       return (
-        <div
-          className={`${interTight.variable} ${robotoMono.variable} dark:bg-slate-940 bg-white font-sans`}
-        >
-          <div id="app" />
-          <div id="modals" />
-          <Story />
-        </div>
+        <TooltipProvider>
+          <div className={`${interTight.variable} ${robotoMono.variable} font-sans`}>
```

Comment:
Do we need `w-screen` to avoid squashing stories to the min width?
<img width="1016" alt="image" src="https://github.com/inngest/inngest/assets/20100586/610cb995-ce8f-477a-9330-971d716bfb26">